common/alert/reports/a11y-1.json

```diff
@@ -0,0 +1,908 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/alert/en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "",
+    "earl:pointer": {
+      "@type": "",
+      "@value": ""
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Service Canada - Principal Publisher",
+      "foaf:homepage": "https://github.com/Shiva-sc",
+      "@type": ["earl:Assertor", "foaf:Person"],
+      "foaf:member": {
+        "@id": "_:accessibility_insights",
+        "doap:name": "Accessibility Insights for Web",
+        "doap:homepage": "https://accessibilityinsights.io/",
+        "doap:repository": "https://github.com/microsoft/accessibility-insights-web",
+        "doap:created": "2022-09-30",
+        "doap:release": "v2.35.0",
+        "@type": ["earl:Assertor", "earl:Software"]
+      }
+    }
+  },
+
+  "dct:date": "2022-02-23",
+
+  "dct:description": "Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2",
+
+  "acr:standard": "https://www.w3.org/TR/WCAG21",
+  "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+  "earl:result": [
+    {
+      "@id": "_:assessment_tool_result",
+      "earl:test": "act_rules:wcag2x/automated/accessibility_insights_web",
+      "earl:subject": "_:subject",
+      "earl:assertedBy": "_:accessibility_insights",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "earl:mode": "earl:manual",
+      "dct:source": {
+        "@type": "acr:Asset",
+        "@value": ""
+      },
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
```

Comment:
Do you have the JSON report that was produced by the accessibility_insights_web? if so can you add as an asset it like this: https://github.com/wet-boew/GCWeb/blob/master/sites/footers/reports/a11y-1.json#L56-L63